tasks1-10/task4.rb

```diff
@@ -0,0 +1,24 @@
+class Four
+  def largest_palindrome_3_digits
+    boundary = 999
```

Comment:
make boundary and lowest variables to be constants and move them out of function